### PR TITLE
feature/train-steam

### PR DESCRIPTION
**Pull request by JoepEilander**

### CODE REVIEW

File: Assets/Art/2D/Sprites/Train/SH_Steam_512x512.png
Comment (lucasvdiepen) [inline Assets/Art/2D/Sprites/Train/SH_Steam_512x512.png:1]: Remove `512x512` in name.

File: Assets/Prefabs/Train/TrainController/Steam_ParticleSystem.prefab
Comment (lucasvdiepen) [inline Assets/Prefabs/Train/TrainController/Steam_ParticleSystem.prefab:1]: Change name to `SteamParticleEffect`.

File: Assets/Prefabs/Train/TrainController/Train.prefab
Comment (lucasvdiepen) [inline Assets/Prefabs/Train/TrainController/Train.prefab:1]: Changes should have been applied to the front cabin prefab.

**Review by lucasvdiepen — CHANGES_REQUESTED** [review]

**Review by Feetjexx — APPROVED** [review]
Approved

**Review by PiterGroot — APPROVED** [review]

**Review by lucasvdiepen — APPROVED** [review]

**Pull request merged**